src/resources/elements/formattedNumber/formattedNumber.ts

```diff
@@ -54,7 +54,9 @@ export class FormattedNumber {
   }
 
   @computedFrom("_value")
-  private get tooltip():string {
-    return this._value?.toString(10);
+  private get tooltip(): string {
+    const withCommasValueConverter = this.container.get(WithCommasValueConverter);
+    const formatted = withCommasValueConverter.toView(this._value?.toString(10));
+    return formatted;
```

Comment:
https://github.com/PrimeDAO/prime-deals-dapp/issues/357#issuecomment-1067391004
> I would prefer to not refactor code to make it be the default. You can supply | withCommas to formattedNumber.

In that case, can you help me with the below code?
I can't get that to work with the `ethwei` value converter

```html
<formatted-number
  if.bind="token.instantTransferAmount"
  thousands-separated
  value.to-view="token.instantTransferAmount | ethwei | withCommas"      <---- here
></formatted-number>
```

Reply:
A bit of debugging shows, that there may be a big in `FormattedNumber.ts`
Somewhere in this if-check
https://github.com/PrimeDAO/prime-deals-dapp/blob/52ec5ee001e77e9f00260b4dc94783f8ba637237/src/resources/elements/formattedNumber/formattedNumber.ts#L41

![image](https://user-images.githubusercontent.com/30693990/158362621-e37adf71-5992-417a-97f3-cbdba87d9c44.png)